lib/spack/spack/schema/bootstrap.py

```diff
@@ -12,6 +25,10 @@
             'root': {
                 'type': 'string'
             },
+            'sources': {
+                'type': 'array',
+                'items': _source_schema
+            }
```

Comment:
I think this is missing the schema for `trusted`

Reply:
Fixed in 156166e